### PR TITLE
70823618 - scss variables

### PR DESCRIPTION
**Pull request by catherinsmi**
Task: https://github.com/users/MariiaBel/projects/3/views/1?pane=issue&itemId=70823618

This PR:
1) adds css variables and global styles
2) adds color list in storybook

In the file stories/colors.js, we get an array of colors from custom variables, create markup for the card, and create a function to generate a palette of custom colors

### CODE REVIEW

**Comment by MariiaBel** [comment]
Do you need storybook-static folder?

File: src/app/styles/global.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+.body {
+    font-family: var(--ff-primary);
+    font-size: var(--fs-base);
+    font-weight: 400;
+    margin: 0;
+}
+
+.test {
+    color: rgba(var(--clr-gray-34), 0.5)
+}
```
Comment (MariiaBel) [inline src/app/styles/global.scss:12]: Delete it please

Comment (catherinsmi) [reply src/app/styles/global.scss:12]: Done

File: src/stories/colors/colors.stories.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { createColors } from "./colors";
+import './colors.scss';
+
+export default {
+    title: "Component/Colors",
```
[the remark this answers is not in the document]
Comment (catherinsmi) [reply src/stories/colors/colors.stories.js]: Done

File: src/app/styles/variables.scss [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+--clr-primary-black: #222021; 
+--clr-primary-gray: #252629;
+//   Цветовая тема для сайд бар
+--clr-secondary-gray: #2A2C30; 
+--clr-selected: #342148; 
+//   Цветовая тема для иконок
+--clr-dark-gray: #4D4E51; 
+--clr-light-gray: #D6D5D5; 
+--clr-pale-gray: #ACACAC; 
+//   Цветовая тема для типографики
```
[the remark this answers is not in the document]
Comment (catherinsmi) [reply src/app/styles/variables.scss]: Ok, but I have difficulties choosing a name. I took these variable names from here: https://stackoverflow.com/questions/50910404/best-practice-for-naming-css-color-variables

**Review by MariiaBel — APPROVED** [review]

**Pull request merged**